kubetest2-tf/deployer/deployer.go

```diff
@@ -106,6 +112,12 @@ func bindFlags(d *deployer) *pflag.FlagSet {
 	flags.BoolVar(
 		&breakKubetestOnUpFail, "break-kubetest-on-upfail", false, "Breaks kubetest2 when up fails",
 	)
+	flags.StringArrayVar(
```

Comment:
explore `StringToStringVarP` function which does the key-value pair mapping automatically from the command line.

Reply:
Below worked after changing to `StringToStringVar`
```
 --extra-vars "build_version=v1.19.2,runtime=containerd,bucket=kubernetes-release,directory=release,release_marker=release/v1.19.2"
```